### ISSUE TITLE
Reopen #6109  

### ISSUE THREAD

**Issue by arbadacarbaYK** [BlueWallet/BlueWallet#6123]
Fix #6109  from yesterday worked - until today

**Comment by arbadacarbaYK** [comment]
Pushing cause i need my accounts back i cannot login to now :(

**Comment by Overtorment** [comment]
can you elaborate? i see zero crashes logged:

![image](https://github.com/BlueWallet/BlueWallet/assets/1913337/607cf688-5af2-42ab-abb1-cdfcf827463b)

**Comment by Overtorment** [comment]
your userid havent changed?

**Comment by arbadacarbaYK** [comment]
youre right, it started working again yesterday and didnt close the issue 😅 no idea what happened inbetween but happy for now to have the accounts back 😂 thx!

**Comment by Overtorment** [comment]
we had electrum server maitenance, it was turned off, but the app should have fellback on other servers in the list.
if its crashing in such scenario that is not alright, but ill take a look another time.

**Comment by arbadacarbaYK** [comment]
![Screenshot_2024-02-26-11-49-36-465_io.bluewallet.bluewallet.jpg](https://github.com/BlueWallet/BlueWallet/assets/63317640/64aef575-4f99-464b-88cc-69d7f61a5529)

Stopped working again 😑

**Comment by arbadacarbaYK** [comment]
so the login doesnt work again, no black screen, no crashing, the login just doesnt work. Quite annoying.

**Comment by Overtorment** [comment]
ok, what are the steps to reproduce?
i need LDK wallet, right? then what?
